### ISSUE TITLE
Issue for go to date in ISO

### ISSUE THREAD

**Issue by npt05** [tradingview/charting-library-examples#396]
https://github.com/user-attachments/assets/af68b16b-b1d0-435e-b6e4-80cb3c675c18
Please help take a look video.
I am developing a website that loads charts using React js, I am using go to date function to customize the time range
I have included it in my app in iframe,
The go to date function to change the time range works on web and android but not on ISO
What is going wrong? How can I fix it ?

My advanced chartLibrary Version: 28

**Comment by romfrancois** [comment]
Do you see any errors in the console or somewhere? Nothing obvious looking at the video. It seems that something is closing and/or preventing the values to be changed though.
Weird it's only happening on iOS and not elsewhere.

**Comment by npt05** [comment]
> Do you see any errors in the console or somewhere? Nothing obvious looking at the video. It seems that something is closing and/or preventing the values to be changed though. Weird it's only happening on iOS and not elsewhere.

Yes, It just happen in ISO, you can see in video it work in web but when we include web view in IOS it not working

https://github.com/user-attachments/assets/a7ef7063-93a3-4cbe-805f-5d63c06f59df
